marklogic-data-hub/src/test/java/com/marklogic/hub/flow/AddSourceNameAndTypeStepDefTest.java

```diff
@@ -0,0 +1,124 @@
+package com.marklogic.hub.flow;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.node.ArrayNode;
+import com.marklogic.hub.AbstractHubCoreTest;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.w3c.dom.Document;
+import org.w3c.dom.NodeList;
+
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.Map;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+public class AddSourceNameAndTypeStepDefTest extends AbstractHubCoreTest {
+    @BeforeEach
+    void beforeEach() {
+        installProjectInFolder("test-projects/addSources-step-definition-test");
+        String path = "test-projects/addSources-step-definition-test/data";
+        FlowInputs inputs = new FlowInputs("addSourcesFlow", "1");
+        inputs.setInputFilePath(getClass().getClassLoader().getResource(path).getPath());
```

Comment:
You should be able to use "makeInputFilePathsAbsoluteInFlow" here. 

Pinging @akshaysonvane  about this - I am wondering though if the best way to override this in tests is via setInputFilePath. But it should be a lot easier than this when there's only one ingestion step (the above makeInputfilePaths function is useful when there are multiple ingestion steps, and each needs it's one absolute path). How about:

    addAbsoluteInputFilePath(flowInputs, "data");

That would use the current HubProject to resolve the "data" directory and then add the absolute file path of that directory to flowInputs.

Reply:
Added the method addAbsoluteInputFilePath in AbstractHubTest class which reads flowInputs and input filePath and adds the absolute path of the directory to flowInputs